readthedocs/search/tests/test_api.py

```diff
@@ -18,12 +18,11 @@
 @pytest.mark.search
 class TestDocumentSearch:
 
-    @classmethod
-    def setup_class(cls):
-        # This reverse needs to be inside the ``setup_class`` method because from
+    def setup_method(self, method):
+        # This reverse needs to be inside the ``setup_method`` method because from
         # the Corporate site we don't define this URL if ``-ext`` module is not
         # installed
-        cls.url = reverse('doc_search')
+        self.url = reverse('doc_search')
```

Comment:
Tests were failing bc the tests for proxied api were using `doc_search` from `readthedocs.urls` for both classes, instead of evaluating it for each class.